File: demo_laya_js/src/ui/LetoMainDemo.js
export default class LetoMainDemo extends Laya.Scene {
    constructor() {
        super();
        
        //加载场景文件
        this.loadScene("test/LetoMainDemo.scene");

        // event
        this.btnRewardedVideo.on(Laya.Event.CLICK, this, this.onRewardedVideoClicked)
        this.btnBanner.on(Laya.Event.CLICK, this, this.onBannerClicked)
        this.btnFeed.on(Laya.Event.CLICK, this, this.onFeedClicked)
        this.btnInterstitial.on(Laya.Event.CLICK, this, this.onInterstitialClicked)
        this.btnFullVideo.on(Laya.Event.CLICK, this, this.onFullVideoClicked)
    }

    onAwake() {
        LTJSSDK.setLogDebug(true)
    }

    onRewardedVideoClicked() {
        Laya.Scene.open('test/LetoRewardedVideoDemo.scene')
    }

    onBannerClicked() {
        Laya.Scene.open('test/LetoBannerDemo.scene')
    }

    onFeedClicked() {
        Laya.Scene.open('test/LetoFeedDemo.scene')
    }

    onInterstitialClicked() {
        Laya.Scene.open('test/LetoInterstitialDemo.scene')
    }

    onFullVideoClicked() {
        Laya.Scene.open('test/LetoFullVideoDemo.scene')
    }
}

File: demo_unity/Assets/Scripts/LetoMainDemo.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using LetoAd;
using UnityEngine.SceneManagement;

public class LetoMainDemo : MonoBehaviour {
    private ILTCSSDK _letoSdk;

    void Awake() {
        _letoSdk = new LTCSSDK();
        _letoSdk.initSDK();
    }

    // Start is called before the first frame update
    void Start()
    {
        
    }

    // Update is called once per frame
    void Update()
    {
        
    }

    public void OnRewardedVideoClicked() {
        SceneManager.LoadScene("LetoRewardedVideoDemo");
    }

    public void OnInterstitialClicked() {
        SceneManager.LoadScene("LetoInterstitialDemo");
    }

    public void OnBannerClicked() {
        SceneManager.LoadScene("LetoBannerDemo");
    }

    public void OnFeedClicked() {
        SceneManager.LoadScene("LetoFeedDemo");
    }

    public void OnFullVideoClicked() {
        SceneManager.LoadScene("LetoFullVideoDemo");
    }
}
